src/Sarif/Readers/SarifContractResolver.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Readers
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif")]
```

Comment:
I just looked at the GlobalSuppressions.cs file and you already have the global suppression for "Sarif". Try removing one of the individual suppressions and see if the warning is still suppressed. If so, get rid of all the individual suppressions for "Sarif". Try the lower-case "sarif" suppressions separately and add global suppression if needed.

---
In reply to: [66075163](https://github.com/Microsoft/sarif-sdk/pull/339#discussion_r66075163) [](ancestors = 66075163)